perception/shape_estimation/README.md

```diff
@@ -38,8 +38,8 @@ This node calculates a refined object shape (bounding box, cylinder, convex hull
 
 | Name                        | Type | Default Value | Description                                         |
 | --------------------------- | ---- | ------------- | --------------------------------------------------- |
-| `use_corrector`             | bool | true          | The flag to apply rule-based filter                 |
-| `use_filter`                | bool | true          | The flag to apply rule-based corrector              |
+| `use_corrector`             | bool | true          | The flag to apply rule-based corrector              |
+| `use_filter`                | bool | true          | The flag to apply rule-based filter                 |
 | `use_vehicle_reference_yaw` | bool | true          | The flag to use vehicle reference yaw for corrector |
```

Comment:
you can now make use of auto-generation of parameter table in the documentation (https://github.com/autowarefoundation/autoware.universe/commit/63033beebc9d45b6185b8ec789f41a7b379ac04a#diff-f5557a723bf6eba2f59cb3ac58f670f0ad7ea7f189fa7c868289ffbc2ab574d9)

Reply:
Hi @ambroise-arm, 
Do you mean to change the parameter table to the format below?
![image](https://github.com/autowarefoundation/autoware.universe/assets/36169304/44b1ebb7-cbf8-437f-8aa2-a6044fd278ec)
I am wondering if is it fine to show this way (screenshot below) in the readme file rather than the table in https://autowarefoundation.github.io/autoware.universe/main/perception/lidar_apollo_segmentation_tvm_nodes/#parameters. 
![image](https://github.com/autowarefoundation/autoware.universe/assets/36169304/bf3dca7b-c72a-444b-8ba9-dc8602b2d0b0)

Reply:
Got it! Done!